### PR TITLE
[Application] yolo example moving weight from torch to nntrainer

### PR DESCRIPTION
**Pull request by baek2sm**
Add yolo example moving weight from pytorch to nntrainer.

Signed-off-by: Seungbaek Hong [sb92.hong@samsung.com](mailto:sb92.hong@samsung.com)

### CODE REVIEW

File: tools/pyutils/torchconverter.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+##
+# Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
+#
+# @file transLayer_v2.py
```
Comment (jijoongmoon) [inline tools/pyutils/torchconverter.py:6]: transLayer_v2.py?

Comment (baek2sm) [reply tools/pyutils/torchconverter.py]: It reuse torchconverter.py in pyutils. thanks!

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by songgot — APPROVED** [review]
LGTM

**Review by DonghakPark — APPROVED** [review]
LGTM! 
Please Check conflicts

**Pull request merged**